concent_api/utils/helpers.py

```diff
@@ -159,3 +159,10 @@ def calculate_maximum_download_time(size: int) -> int:
             settings.MINIMUM_UPLOAD_RATE,
         ).total_seconds()
     )
+
+
+def join_messages(*msg):
```

Comment:
1) Please avoid using shortcuts. `message` whould be a better name.
2) This is a collection so it should be `messages`.